Backend/RoomMe/RoomMe.API/Converters/HouseworkConverter.cs

```diff
@@ -0,0 +1,100 @@
+﻿using RoomMe.API.Models;
+using RoomMe.SQLContext.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace RoomMe.API.Converters
+{
+    public static class HouseworkConverter
+    {
+        public static HouseworkFullGetModel ToHouseworkFullModel(this Housework housework)
+        {
+            return new HouseworkFullGetModel
+            {
+                Id = housework.Id,
+                Name = housework.Name,
+                Flat = housework.Flat.ToFlatNameModel(),
+                Author = housework.Author.ToUserNicknameModel(),
+                Description = housework.Description,
+                Users = housework.Users.Select(x => x.ToUserNicknameModel()).ToList(),
+                Schedules = housework.HouseworkSchedules.Select(x => x.ToScheduleDateModel()).ToList()
+            };
+        }
+
+        public static Housework ToHouseworkModel(this HouseworkPutModel housework)
+        {
+            return new Housework()
+            {
+                Name = housework.Name,
+                FlatId = housework.FlatId,
+                //TODO: When JWT authorization is implemented this should be received through the token
+                AuthorId = 1,
```

Comment:
Mamy autoryzacje juz - za pomoca sessionHelpera bedziesz mogl to wyciagnac po zmergowaniu deva